GitUI/Theming/ThemePersistence.cs

```diff
@@ -159,24 +137,12 @@ private bool TryReadFile(string fileName, out string result, bool quiet)
             }
             catch (Exception ex)
             {
-                WarnOnInvalidContent(fileName, ex.Message, quiet);
+                PrintTraceWarning(fileName, ex.Message);
                 return false;
             }
         }
 
-        private void WarnOnInvalidContent(string fileName, string message, bool quiet)
-        {
-            var fullMessage = string.Format(_failedToLoadThemeFrom.Text, fileName) +
-                Environment.NewLine + message;
-
-            Trace.WriteLine(fullMessage);
-            if (!quiet)
-            {
-                MessageBox.Show(
-                    message, _failedToLoadTheme.Text,
-                    MessageBoxButtons.OK,
-                    MessageBoxIcon.Exclamation);
-            }
-        }
+        private void PrintTraceWarning(string fileName, string message) =>
+            Trace.WriteLine(string.Format(_failedToLoadThemeFrom.Text, fileName) + Environment.NewLine + message);
```

Comment:
Do we need this in Release builds? If not, let's decorate the method with `[Conditional("DEBUG")]`

Reply:
ok